cmd/relay-s3/main.go

```diff
@@ -0,0 +1,198 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
how big of a dependency are the aws client libraries? if it's big, we may want a separate go.mod to allow the rest of the project to be built without needing to download them.

Reply:
There’s a _lot_ of bad documentation. I assume it’s quite big but I do not know numbers. How do you normally get this measurement in golang?